pkgs/misc/urbit/default.nix

```diff
@@ -1,21 +1,21 @@
 { stdenv, fetchFromGitHub, gcc, gmp, libsigsegv, openssl, automake, autoconf, ragel,
-  cmake, re2c, libtool, ncurses, perl, zlib, python2 }:
+  cmake, re2c, libtool, ncurses, perl, zlib, python2, curl }:
 
 stdenv.mkDerivation rec {
 
   name = "urbit-${version}";
-  version = "2016-06-02";
+  version = "2016-11-01";
```

Comment:
There seems to be a reasonable released version of this tool: https://github.com/urbit/urbit/releases/tag/v0.4